src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -226,6 +276,7 @@ def acr_repository_show_tags(registry_name,
         registry_name=registry_name,
         resource_group_name=resource_group_name,
         path='/v2/{}/tags/list'.format(repository),
+        permission='pull',
```

Comment:
one question: for CLI to send request to registry, does it first get a challenge response from registry? If we do, there is information in the challenge response on what permission is needed. If we are not getting challenge response but hard-code the permission, there is a concern that this may change from the server side (although unlikely, just keep in mind).